src/base/install/installDevTools.m

```diff
@@ -99,7 +106,7 @@ function installDevTools()
             % removing .git folder
             rmpath(genpath([devToolsDir filesep '.git']));
 
-            fprintf([' > In order to start contributing, you can now run:\n\n >> contribute\n\n']);
+            fprintf([' > In order to start contributing, you can now run:\n\n >> contribute(''' repoName '''); \n\n']);
```

Comment:
One question:
Will it be necessary to always indicate the repoName when using contribute? Or will this (by default) be detected automatically based on the git information in the current folder?

Reply:
It is defined as follows:
```
>> contribute
```
is the same as
```
>> contribute('opencobra/cobratoolbox')
```
When contributing to any other repo:
```
>> contribute('orgName/repoName')
```